doc/en/components/inputs/combo/templates.md

```diff
@@ -120,6 +120,14 @@ To render a custom header above the list of options pass content to the `header`
 </IgbCombo>
 ```
 
```

Comment:
Not part of the changeset so cannot reference the specific lines but the snippets for the item template and the group header template appear to be for web components.